news/change_07-09-17.md

```diff
@@ -1,6 +1,7 @@
 ### Project: [Change](../projects/change.md) [Indorse](../projects/indorse.md)
 ### Date: 07 September 2017
 ## Summary
+  
 Indorse CEO, David Moskowitz, had announced a partnership with Change, an upcoming decentralised crypto bank.
```

Comment:
https://github.com/NrgDumik/base/blob/b094250c3666cf1d45aeb652fa21f210198f2cc3/news/change_07-09-17.md 
no paragraphs: 
decentralised crypto bank. [double space]
...of applicants for the Change marketplace.[double space]